Pokemon Unity/Assets/Scripts2/Pokemon/Pokemon.cs

```diff
@@ -269,6 +265,70 @@ public Pokemon(Pokemons pokemon) : this()
 
             //calcStats();
         }
+
+        public Pokemon(string nickName, int form,
+            Pokemons species, Abilities ability,
+            Nature nature,
+            bool isShiny, bool? gender,
+            int[] pokerus, int pokerusStrain,
+            int? shadowLevel,
+            int currentHp, Items item,
+            byte[] iv, byte[] ev, 
+            int obtainedLevel, int currentLevel, int currentExp,
+            int happiness, Status status, int statusCount,
+            int eggSteps, Items ballUsed,
+            Item.Item.Mail mail, Move[] moves,
+            Ribbon[] ribbons, bool[] markings,
+            int personalId,
+            ObtainedMethod obtainedMethod,
+            DateTimeOffset timeReceived, DateTimeOffset timeEggHatched) : this()
+        {
+            name = nickName;
+            Form = form;
+            _base = new PokemonData(species);
```

Comment:
this part is automatic, if you use a constructor that inherits from previous constructor

Reply:
look @ line 284 => `: this()`

Reply:
Now that you corrected line 284, you should comment out this line here (297)